packages/calypso-products/src/constants/jetpack.ts

```diff
@@ -800,6 +803,12 @@ export const JETPACK_PRODUCT_RECCOMENDATION_MAP: Record< string, string[] > = {
 		JETPACK_TAG_FOR_ONLINE_FORUMS,
 		JETPACK_TAG_FOR_WOOCOMMERCE_STORES,
 	],
+	[ PRODUCT_JETPACK_CREATOR_BI_YEARLY ]: [
```

Comment:
Looks like this needs to be: `PRODUCT_JETPACK_CREATOR_BI_YEARLY`

```suggestion
	[ PRODUCT_JETPACK_CREATOR_YEARLY ]: [
```